src/main/java/wolox/training/models/User.java

```diff
@@ -39,6 +39,19 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
+    @Column(nullable = false)
+    private String password;
+
+    public void setPassword(String password){
+        this.password = checkString(password, "password");
+    }
+
+    @Column(nullable = false)
+    private String role = "USER";
+    public void setRole(String role){
+        this.role = checkString(role, "role");
+    }
+
```

Comment:
Remember that the methods are after all the member declarations (In line 80 or so)
If the setters are used only in this class, It's better to have them as private methods